report/renderer/details-renderer.js

```diff
@@ -394,29 +496,48 @@ export class DetailsRenderer {
       this._dom.createChildOf(theadTrElem, 'th', classes).append(labelEl);
     }
 
+    const entityItems = this._getEntityGroupItems(details);
     const tbodyElem = this._dom.createChildOf(tableElem, 'tbody');
-    let even = true;
-    for (const item of details.items) {
-      const rowsFragment = this._renderTableRowsFromItem(item, details.headings);
-
-      // The attribute item.entity could be a string (entity-classification), or
-      // a LinkValue for ThirdPartySummary audit.
-      let entityName;
-      if (typeof item.entity === 'object' && item.entity.type === 'link') {
-        entityName = item.entity.text;
-      } else if (typeof item.entity === 'string') {
-        entityName = item.entity;
+    if (entityItems.length) {
+      for (const entityItem of entityItems) {
+        const entityName = typeof entityItem.entity === 'string' ? entityItem.entity : undefined;
+        // Render the heading row
+        const entityGroupFragment = this._renderEntityGroupRow(entityItem, details.headings);
+        // Render all the items that match the heading row
+        for (const item of details.items.filter((item) => item.entity === entityName)) {
+          entityGroupFragment.append(this._renderTableRowsFromItem(item, details.headings));
+        }
+        const rowEls = this._dom.findAll('tr', entityGroupFragment);
+        const firstRowEl = rowEls[0];
+        firstRowEl.classList.add('lh-row--group');
```

Comment:
Could this be done in `_renderEntityGroupRow`?

Reply:
Yep.